NetKAN/OPTSpacePlaneMain.netkan

```diff
@@ -0,0 +1,33 @@
+{
+    "spec_version" : "v1.9",
+    "identifier"   : "OPTSpacePlaneMain",
+    "$kref"        : "#/ckan/curse/225018",
+    "license"      : "CC-BY-NC-SA-4.0",
+    "abstract"     : "Over 50 stock-a-like, large space plane oriented parts",
+    "resources"    : {
+        "homepage": "http://forum.kerbalspaceprogram.com/index.php?/topic/151118-120-opt-space-plane-v190/"
+    },
+    "ksp_version": "1.2.0",
+    "depends": [
+        { "name" : "FirespitterCore" },
+        { "name" : "ModuleManager" }
+    ],
+    "suggests": [
+        { "name": "RasterPropMonitor" },
+        { "name": "ASETAvionics" },
+        { "name": "ASETProps" }
```

Comment:
It may have been accidentally included. The image marks it as "other mod" like DistantObject and Planetshine. I'll include it, though. It's only a suggests.